packages/@sanity/form-builder/src/inputs/PortableText/Editor.tsx

```diff
@@ -175,6 +175,14 @@ export function Editor(props: EditorProps) {
     [scrollElement]
   )
 
+  const spellCheck = useMemo(() => {
+    // Only enable from schema
+    if (editor.portableTextFeatures.types.block.options?.spellCheck) {
+      return true
+    }
+    return false
```

Comment:
Do we not want to enable/disable this based on user agent, or is that too inconsistent?

```suggestion
  const spellCheck = editor.portableTextFeatures.types.block.options?.spellCheck

  // Allow the user to explicitly opt-in/opt-out of spellchecking
  if (typeof spellCheck !== 'undefined') {
    return Boolean(spellCheck)
  }

  // Chrome 96+ has a severe performance regression when using spell checking for contentEditable fields
  return typeof navigator === 'undefined' ? true : !/Chrome\/96/.test(navigator.userAgent)
```

Maybe the regex should be `(9\d|1\d\d)` or something until the issue is fixed, but you get the gist. Thoughts?

Reply:
I was thinking to draw to birds with one stone here. We have been planning to make spellCheck configurable via the schema, and when this is the case, the default should probably be "off". If you want spellCheck then enable it.

spellChecking works with various success both because of language mismatches, and it can also be a performance hog so I'm in favour of putting it off by default.

If this is not what we want, and we want default on, then we def. should do the browser version check.

Reply:
I think this should be allowed to return `undefined` so that it will use the browser default by default.